pages/ambassadors/index.tsx

```diff
@@ -0,0 +1,573 @@
+            opacity="0.8"
+          />
+          <Box
+            h="1px"
+            bg="linear-gradient(93.96deg, rgba(25, 26, 27, 0.8) 17.14%, rgba(24, 67, 78, 0.8) 36.78%, rgba(108, 47, 115, 0.8) 61%, rgba(25, 26, 27, 0.8) 79.98%)"
+            opacity="0.8"
+          />
+        </Box>
+        <HomepageSection id="get-started" bottomPattern>
+          <Flex
+            flexDir="column"
+            pt={{ base: 12, lg: 24 }}
+            pb={{ base: 24, lg: 0 }}
+            align="center"
+            gap={{ base: 6, md: 8 }}
+          >
+            <Center mb={6} pt={{ base: 8, lg: 24 }}>
+              <Center p={2} position="relative" mb={6}>
+                <Box
+                  position="absolute"
+                  bgGradient="linear(to-r, #F213A4, #040BBF)"
+                  top={0}
+                  left={0}
+                  bottom={0}
+                  right={0}
+                  borderRadius="3xl"
+                  overflow="visible"
+                  filter="blur(15px)"
+                />
+
+                <ChakraNextImage
+                  alt=""
+                  boxSize={{ base: 24, md: 32 }}
+                  placeholder="empty"
+                  src={WhiteLogo}
+                />
+              </Center>
+            </Center>
+            <Heading as="h2" size="display.md" textAlign="center">
+              Ready to be one of us?
+            </Heading>
+            <LightMode>
+              <LinkButton
+                role="group"
+                leftIcon={
+                  <Icon
+                    as={BsLightningCharge}
+                    color="#1D64EF"
+                    transitionDuration="slow"
+                    transitionTimingFunction="easeOut"
+                    _groupHover={{ color: "#E0507A" }}
+                  />
+                }
+                color="black"
+                px={20}
+                py={{ base: 5, md: 7 }}
+                onClick={() =>
+                  trackEvent({
+                    category: "cta-button",
+                    action: "click",
+                    label: "start",
+                    title: "Join the waitlist",
+                  })
+                }
+                textAlign="center"
+                variant="gradient"
+                fromcolor="#1D64EF"
+                tocolor="#E0507A"
+                size="lg"
+                borderRadius="md"
+                href="https://docs.google.com/forms/d/e/1FAIpQLSfuhnYtA-CMo5PT1jF4vzzPy0jVS_DNRMSOayV4FNxx7rpKKQ/viewform"
+              >
+                <Box as="span" py={0.5}>
+                  Join the Mission
+                </Box>
+              </LinkButton>
+            </LightMode>
+          </Flex>
+        </HomepageSection>
+        <NewsletterSection />
+        <HomepageFooter />
+      </Flex>
+    </DarkMode>
+  );
+};
+
+Ambassadors.pageId = PageId.Learn;
```

Comment:
Ambassadors.pageId = PageId.Ambassadors

Reply:
@nachoiacovino If you merge just this /ambassadors/index.tsx it can work? With the related assets

Reply:
I can't nitpick your PR unless I literally copy the changes and push them myself which will take your commits away, do you think you could fix the things I said so I can deploy and check if there's any more changes needed?